client/src/components/fields/RenderRadioGroup.js

```diff
@@ -43,21 +43,24 @@ export const RenderRadioGroup = ({
     <Fragment>
       {label && <InputFieldLabel label={label} boldLabel={boldLabel} />}
       <RadioGroup {...field} {...props} onChange={handleChange}>
-        {options.map((option) => (
-          <Fragment key={option.value}>
-            <FormControlLabel
-              value={option.value}
-              checked={field.value === option.value}
-              label={option.label}
-              disabled={disabled}
-              labelPlacement='end'
-              control={<Radio color={option.color || 'primary'} />}
-            />
-            {hiddenCheckbox?.fields.includes(field.value) &&
-              option.value === field.value &&
-              hiddenCheckbox.node}
-          </Fragment>
-        ))}
+        {options.map(
+          (option) =>
+            option && (
```

Comment:
What is the purpose of this check? Why might `options` contain a falsy value?

Reply:
In ManageGraduationForm.js I conditionally rendered the options depending on if bulk graduation was occurring (We only want to allow the successful flow) 
```
 options={[
                    {
                      value: postHireStatuses.postSecondaryEducationCompleted,
                      label: 'Graduated',
                    },
                    !isBulkGraduate && {
                      value: postHireStatuses.cohortUnsuccessful,
                      label: 'Unsuccessful cohort',
                    },
                  ]}
```
This returned options as [object, false], which was displaying the radio button without anything else.

Reply:
In that case, I'd say you're better off filtering out the potentially false element there (`.filter(Boolean)` on the array as you define it) instead of making this logic expect arbitrary falsy values mixed in with actual options.

Reply:
Updated!